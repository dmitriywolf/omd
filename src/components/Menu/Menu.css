.menu {
  display: flex;
  align-items: center;
}

.menu__link {
  margin: 0 10px;
  font-size: 20px;
  color: grey;
  text-transform: uppercase;
}

.menu__link:hover {
  color: #ffffff;
}

.menu__link._active {
  color: white;
}
