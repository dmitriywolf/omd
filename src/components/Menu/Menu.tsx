import React from 'react';
import {NavLink} from 'react-router-dom';
import './Menu.css';

const Menu = () => {
  return (
    <div className="menu">
      <NavLink exact to="/" className="menu__link" activeClassName="_active">Home</NavLink>
      <NavLink to="/about" className="menu__link" activeClassName="_active">About team</NavLink>
    </div>
  )
};

export default Menu;
