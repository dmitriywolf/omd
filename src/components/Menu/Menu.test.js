import React from 'react';
import Menu from './Menu';
import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";

describe("Menu component:", () => {
  it("should render Menu component", () => {
    const component = shallow(<Menu/>);
    expect(toJson(component)).toMatchSnapshot();
  })
});
