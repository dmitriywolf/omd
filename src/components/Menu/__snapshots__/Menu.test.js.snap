// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component: should render Menu component 1`] = `
<div
  className="menu"
>
  <NavLink
    activeClassName="_active"
    className="menu__link"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="_active"
    className="menu__link"
    to="/about"
  >
    About team
  </NavLink>
</div>
`;
